source/_integrations/somfy.markdown

```diff
@@ -58,8 +58,15 @@ client_secret:
   description: Your Somfy consumer secret.
   required: true
   type: string
+optimistic:
+  description: Set optimistic mode.
+  required: false
+  default: False
+  type: boolean
 {% endconfiguration %}
 
+**optimistic** mode should only be used when the integration is not able to gain information on whether a cover is open or closed. It will attempt to track the status within Home Assistant. This mode should only be used if Home Assistant is the only way you operate the blind. If you also use the physcial remote control or the Somfy app, Home Assistant will become out of sync.
```

Comment:
```suggestion
**optimistic** mode should only be used when the integration is not able to gain information on whether a cover is open or closed. It will attempt to track the status within Home Assistant. This mode should only be used if Home Assistant is the only way you operate the blind. If you also use the physical remote control or the Somfy app, Home Assistant will become out of sync.
```